Permissions and SQL elastic pool

kasparks opened the pull request:
It is no longer required for an Owner to visit Advisor first to register
- removed note
- updated FAQ

Removed SQL Elastic Pool cost recommendation
- removed SQL server from resources listed in FAQ question "What resources does Advisor provide recommendations for?"

azurecla commented:
Hi __@kuniteru__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

cjgronlund commented:
@kuniteru 
Thank you for your contribution. I'll send this to the author for verification. 
--Carolyn

cjgronlund commented:
@sahaj08 @barbkess 
Please review these recommended changes. Let me know whether to accept as is, or whether you'd like to incorporate just part. 
--Carolyn

sahaj08 commented:
These changes are okay, please accept as is.

From: C.J. Gronlund [mailto:notifications@github.com]
Sent: Tuesday, March 15, 2016 9:55 AM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Sahaj Saini <sahajs@microsoft.com>
Subject: Re: [azure-content] Update sql-data-warehouse-get-started-load-with-polybase.md (#5991)


@sahaj08<https://github.com/sahaj08> @barbkess<https://github.com/barbkess>
Please review these recommended changes. Let me know whether to accept as is, or whether you'd like to incorporate just part.
--Carolyn

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub:
https://github.com/Azure/azure-content/pull/5991#issuecomment-196920463

barbkess commented:
Good to go
#sign-off

KaitlynCorazao committed:
Permissions and SQL elastic pool

It is no longer required for an Owner to visit Advisor first to register
- removed note
- updated FAQ

Removed SQL Elastic Pool cost recommendation
- removed SQL server from resources listed in FAQ question "What resources does Advisor provide recommendations for?"
